indigo/indigo/src/main/scala/indigo/shared/scenegraph/Camera.scala

```diff
@@ -65,6 +68,8 @@ object Camera:
   final case class LookAt(target: Point, zoom: Zoom, rotation: Radians) extends Camera:
     val isLookAt: Boolean = true
     val position: Point   = target
+    def topLeft(viewport: GameViewport) = 
+      target - Point(viewport.width/2, viewport.height/2) / zoom.toDouble.toInt
```

Comment:
Don't worry about it this time, but you should install Scalafmt, it'll keep all the code consistently formated.